modules/ROOT/design-async-api.adoc

```diff
@@ -0,0 +1,126 @@
+= AsyncAPI Specifications   
+
+AsyncAPI is a language that describes messaging interfaces. This open-source, industry-standard language is agnostic to any technology. To learn more, see the https://www.asyncapi.com/docs/getting-started[AsyncAPI documentation]. 
+
+You can create or import your AsyncAPI specifications in API Designer and then edit, document, and publish them. 
+
+AsyncAPI makes it easy to work with event-driven architecture by separating the API into three layers:
+
+* Events: Message or data to be shared with other services
+* Channels: Destination of the message to be sent or received
+* Transport: Technology that transports the message, such as RabbitMQ, Kafka, or Anypoint MQ
+
+You can create or import an API specification in API Designer.
+
+To create an AsyncAPI specification in API Designer:
+
+. In Design Center, click *Create New*.
+. Select *New AsyncAPI*.
++
+image::platform/anypoint-create-asyncapi.png[The Create menu with the AsyncAPI option shown]
+
+== Import AsyncAPI Specifications
+
+To import AsyncAPI specifications from Anypoint Exchange or from your filesystem:
```

Comment:
Is this procedure needed? If so, is there already one in the Design Center doc you can reuse in a partial? 
I think for now, just to get this published, it's okay to have this here, but definitely double-check that the UI hasn't changed, since it's been a while since this was published.